packages/back-end/src/services/experiments.ts

```diff
@@ -2812,14 +2812,23 @@ export async function updateExperimentAnalysisSummary({
   const overallTraffic = experimentSnapshot.health?.traffic?.overall;
   const snapshotHealthPower = experimentSnapshot.health?.power;
 
-  const totalUsers = overallTraffic?.variationUnits?.reduce(
-    (acc, a) => acc + a,
-    0
-  );
+  const standardSnapshot =
+    experimentSnapshot.type === "standard" &&
+    experimentSnapshot.analyses?.[0].results.length === 1;
+  const totalUsers = overallTraffic?.variationUnits.length
+    ? overallTraffic.variationUnits.reduce((acc, a) => acc + a, 0)
+    : standardSnapshot
+    ? // fall back to first result for standard snapshots if overall traffic
+      // is missing
+      experimentSnapshot?.analyses?.[0]?.results?.[0]?.variations?.reduce(
+        (acc, a) => acc + a.users,
+        0
+      )
+    : null;
```

Comment:
I believe the type of `totalUsers` will be `number | undefined | null` here.  We might want to coerce it to only have `null` or `undefined` and not both.